Subject: src/format_ext2_kernel.ml: Fix kernel filtering for aarch64 architecture
From: Simon Fischer 1522981+Fischer-Simon@users.noreply.github.com Wed Jul 12 17:10:53 2023 +0200
Date: Wed Jul 12 17:10:53 2023 +0200:
Git: 4b3922feb65150f3423d0877038c5ba6e16d910c

Add appropriate globs for arm based kernels. The file names end in -arm64 but the architecture is named aarch64.
diff --git a/src/format_ext2_kernel.ml b/src/format_ext2_kernel.ml
index 6d2e699..4589552 100644
--- a/src/format_ext2_kernel.ml
+++ b/src/format_ext2_kernel.ml
@@ -187,6 +187,7 @@ and patt_of_cpu host_cpu =
     | "amd64" | "x86_64" -> ["amd64"; "x86_64"]
     | "parisc" | "parisc64" -> ["hppa"; "hppa64"]
     | "ppc64el" -> ["powerpc64le"]
+    | "aarch64" -> ["aarch64"; "arm64"]
     | _ when host_cpu.[0] = 'i' && host_cpu.[2] = '8' && host_cpu.[3] = '6' -> ["?86"]
     | _ when String.length host_cpu >= 5 && String.sub host_cpu 0 5 = "armv7" ->  ["armmp"]
     | _ -> [host_cpu]
